.coafile

```diff
@@ -0,0 +1,41 @@
+[Default]
+ignore = .tox/**, .gitignore, .gitreview, .gitmodules, **/provisioning/roles/**
```

Comment:
coala has [PuppetLintBear] (https://github.com/coala/coala-bears/blob/3cb9b148adc0dda51ac890188b38fd968f6058fd/bears/configfiles/PuppetLintBear.py). We can add that for `files = **/*.pp`. I am currently writing a bear for [Ansible](https://github.com/coala/coala-bears/issues/794) should be updated in the new release.